apps/ledger-live-mobile/src/components/OperationRowDate.tsx

```diff
@@ -15,5 +15,5 @@ export default function OperationRowDate({ date }: Props) {
       }),
     [date, locale],
   );
-  return `${localeTimeString}`;
+  return <>`${localeTimeString}`</>;
```

Comment:
```suggestion
  return <>{localeTimeString}</>;
```

Broken while doing the non reg.
<img width="199" alt="Screenshot 2022-10-25 at 14 16 23" src="https://user-images.githubusercontent.com/44363395/197770881-d786ab9b-32e6-4959-9f78-2504a1c6f0ea.png">

Reply:
Nice catch, thanks! 👍